sdk/networkcloud/Azure.ResourceManager.NetworkCloud/tests/NetworkCloudManagementTestEnvironment.cs

```diff
@@ -92,5 +95,19 @@ public override TokenCredential Credential
                 return _credential;
             }
         }
+
+        public string GetContainerUri()
```

Comment:
This isn't the way to handle sanitization of recorded variables. We should call GetRecordedVariable and pass in the lambda for IsSecret, e.g. https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/ai/Azure.AI.Inference/tests/InferenceClientTestEnvironment.cs#L24